### PR TITLE
WEBEU2 - Sprint Challenge - Advanced CSS - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+- While adaptive website targets devices with a fixed width, fully responsive website targets devices width a max-width or min-width.
```
Comment (maximesalomon) [inline README.md:37]: Please explain more

File: README.md [hunk 2]
```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
 2. Describe what it means to be mobile first vs desktop first.
 
+- Mobile-first approach to responsive layout design implies that the default styles would target mobile devices (smaller screens), while media queries with min-width property are used to craft styles that target bigger screens (tablets and desktop). Desktop-first approach is the reverse of the mobile-first approach; in desktop-first approach, the default styles would target desktops (bigger screens), while media queries with max-width property are use to write styles that target smaller screens (tablets, phones).
```
Comment (maximesalomon) [inline README.md:41]: Perfect!

File: README.md [hunk 3]
```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+- Font size of 62.5% in the root element (html tag) gives a html file, the base font size of 10px calculated from the browser's default font size of 16px. Thus, mapping 1rem to 10px (1rem == 10px); this makes the unit conversion between px and rem very easy and intutive for developers.
```
Comment (maximesalomon) [inline README.md:45]: 👍

File: README.md [hunk 4]
```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
 4. How would you describe preprocessing to someone new to CSS?
 
+- Preprocessing is the art/act of writing styles in a more appealing language with rich features that's not CSS (ex: LESS), then get the written styles translated to pure CSS styles.
```
Comment (maximesalomon) [inline README.md:49]: 👍

File: README.md [hunk 5]
```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
 5. What is your favorite concept in preprocessing? What is the concept that gives you the most trouble?
 
+- My favorite concept is the `Parameteric Mixins`; which made mixins dynamic and very useful in keeping LESS scripts DRY.
```
Comment (maximesalomon) [inline README.md:53]: Any trouble?

File: README.md [hunk 6]
```diff
@@ -94,41 +104,41 @@ _You will know everything is working properly when you see the styles enabled fo
...
 
 ## Stretch Problems
 
 After finishing your required elements, you can push your work further. These goals may or may not be things you have learned in this module but they build on the material you just studied. Time allowing, stretch your limits and see if you can deliver on the following optional goals:
 
-* [ ] Build a page of your choosing from the navigation items. Come up with content and images that fit the theme.
+* [x] Build a page of your choosing from the navigation items. Come up with content and images that fit the theme.
 
-* [ ] Introduce CSS animations to your site.
+* [x] Introduce CSS animations to your site.
```
Comment (maximesalomon) [inline README.md:141]: Great use of README to check your progress! Continue to do so :)

**Review by maximesalomon — COMMENTED** [review]
Hello James :) Please check your dashboard for my overall comment!

PS - Your git commits could be improved a bit --> https://chris.beams.io/posts/git-commit/

File: README.md (hunk 5 above)
Comment (erozonachi) [reply README.md:53]: No trouble 😁

File: README.md (hunk 1 above)
Comment (erozonachi) [reply README.md:37]: Adaptive websites make use of fixed layout width with media queries, that target specific device widths and as such don't scale well on unaccounted device widths.

Responsive websites make use of min-width or max-width layout with media queries, that target ranges of device widths and thus, scale well across all device widths.

**Pull request merged**